packages/meteor-gazelle-createdby/package.js

```diff
@@ -0,0 +1,27 @@
+Package.describe({
+  name: 'meteor-gazelle:createdby',
```

Comment:
meteor-gazelle:astronomy-createdby-behavior

I think this is a more suitable package title